pkg/cmd/config/set_garden.go

```diff
@@ -29,9 +29,21 @@ func NewCmdConfigSetGarden(f util.Factory, ioStreams util.IOStreams) *cobra.Comm
 		},
 	}
 	cmd := &cobra.Command{
-		Use:               "set-garden",
-		Short:             "modify or add Garden to gardenctl configuration",
-		Long:              "Modify or add Garden to gardenctl configuration. E.g. \"gardenctl config set-garden my-garden --kubeconfig ~/.kube/kubeconfig.yaml\" to configure or add a garden with identity 'my-garden'",
+		Use:   "set-garden",
+		Short: "modify or add a Garden to the gardenctl configuration",
+		Long: `Modify or add a Garden to the gardenctl configuration.
+A valid Garden configuration consists of a name (required), kubeconfig path (required), an optional context as well as any number of patterns.
```

Comment:
```suggestion
A valid Garden configuration consists of a name (required), kubeconfig path (required), a context as well as any number of patterns.
```
I think we do not necessarily need to mention that it's optional (as it's not required)